### PR TITLE
Fix MNTOR-1795 - Add ADR for L10N workflow updates

### PR DESCRIPTION
**Pull request by maxxcrawford**
## Summary

This ADR captures a decision to refactor how L10N works on Monitor. 

Links:
- [MNTOR-1795](https://mozilla-hub.atlassian.net/browse/MNTOR-1795)

### CODE REVIEW

File: docs/adr/0006-localization-workflow.md
Comment (maxxcrawford) [inline docs/adr/0006-localization-workflow.md:3]: Update status once approved

**Review by flodolo — CHANGES_REQUESTED** [review]

File: docs/adr/0006-localization-workflow.md
Comment (maxxcrawford) [reply docs/adr/0006-localization-workflow.md]: Removed in 13e494d2a9ff8c05f30ace880a3dbe5256d46e0d

File: docs/adr/0006-localization-workflow.md
Comment (maxxcrawford) [reply docs/adr/0006-localization-workflow.md]: Rewritten in 13e494d2a9ff8c05f30ace880a3dbe5256d46e0d

> With the React refactor, this is a good time to revisit our processes, and determine what works the best for everyone.

File: docs/adr/0006-localization-workflow.md
Comment (rhelmer) [inline docs/adr/0006-localization-workflow.md:16]:
```suggestion
  * This is also a chance to better align with other Mozilla projects.
```

**Review by rhelmer — APPROVED** [review]
lgtm, flod is the expert here so I'll defer to him. Thanks for writing this up!

File: docs/adr/0006-localization-workflow.md
Comment (flodolo) [inline docs/adr/0006-localization-workflow.md]: All the other `seperate` typos are still around.

**Review by flodolo — APPROVED** [review]
Fixed some of the nits I reported in a separate commit, hope that's OK.

**Pull request merged**
